catalog-info.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: backstage.io/v1alpha1
+kind: API
+metadata:
+  name: platform-console-api
+  description: API for creating and deploying applications on the VA Platform.
+  annotations:
+    github.com/project-slug: department-of-veterans-affairs/platform-console-api
+  tags:
+    - ruby
+    - rails
+    - platform-console-api
+spec:
+  type: openapi
+  lifecycle: production
+  owner: platform-console-services
+  definition: | 
+    openapi: "3.0.0"
+    info:
+      version: 1.0.0
+      title: Platform Console API
+      license:
+        name: MIT
+    servers:
+      - url: http://platform-console-api.vfs.va.gov/
+    paths:
+      /v1/teams:
+        get:
+          summary: List of teams
```

Comment:
The definition will eventually point to our openapi docs, but for the time being, the console UI team said that it's ok to just put a placeholder here since the definition key is required.

Reply:
You'll need a fake `responses` also. https://swagger.io/specification/#operation-object

Reply:
@Matth-Ng All set! Could you give this a 👍  if it looks good to go?